src/schedule.js

```diff
@@ -150,13 +151,25 @@ Job.prototype.schedule = function(spec) {
   } catch (err) {
     var type = typeof spec;
     if (type === 'string') {
-      spec = new Date(spec);
+      try {
+        spec = rrulestr(spec);
+      } catch(rruleErr) {
+        //if invalid rrule do nothing
```

Comment:
why swallow error silently?

Reply:
Essentially here I am testing to see if it a valid rrule string, if you pass a date object into the rrulestr function itll throw an error which we just want to ignore as if it is a date we just want to execute the code below